src/go/rpk/pkg/cli/cmd/container/start.go

```diff
@@ -236,26 +229,21 @@ func startCluster(
 	for nodeID := uint(1); nodeID < n; nodeID++ {
 		id := nodeID
 		grp.Go(func() error {
-			kafkaPort, err := vnet.GetFreePort()
-			if err != nil {
-				return err
-			}
-			proxyPort, err := vnet.GetFreePort()
-			if err != nil {
-				return err
-			}
-			schemaRegPort, err := vnet.GetFreePort()
-			if err != nil {
-				return err
-			}
-			rpcPort, err := vnet.GetFreePort()
-			if err != nil {
-				return err
-			}
-			metricsPort, err := vnet.GetFreePort()
-			if err != nil {
-				return err
-			}
+			kafkaPort := ports[0]
+			ports = ports[1:]
+
+			proxyPort := ports[0]
+			ports = ports[1:]
+
+			schemaRegPort := ports[0]
+			ports = ports[1:]
+
+			rpcPort := ports[0]
+			ports = ports[1:]
+
+			metricsPort := ports[0]
+			ports = ports[1:]
+
```

Comment:
This block of code is a logic race since `grp.Go` is ran in a goroutine. Instead, we should capture the nodeID, and then
```go
// above
i := nodeID
// inside
var (
  kafkaPort = ports[0 + 5*i]
  proxyPort = ports[1 + 5*i]
  ...
)
```

Reply:
Uh, thanks !  fixed that.